### PR TITLE
Announce upcoming 1.20 EOL

### PR DESCRIPTION
**Pull request by hanxiaop**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by dhawton** [comment]
Seems a bit early. We usually announce this 2 weeks after the minor release goes out, giving a 30 day notice.

**Comment by hanxiaop** [comment]
> Seems a bit early. We usually announce this 2 weeks after the minor release goes out, giving a 30 day notice.

@dhawton Thanks! I'll revise the date later.

File: content/en/news/support/announcing-1.20-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.20 ends on June 25, 2024
+subtitle: Support Announcement
+description: Upcoming Istio 1.20 end of life announcement.
+publishdate: 2024-05-14
```
Comment (hanxiaop) [inline content/en/news/support/announcing-1.20-eol/index.md:5]:
```suggestion
publishdate: 2024-05-24
```

**Comment by hanxiaop** [comment]
@dhawton I think we can merge this on Friday, and there will be one month left for the EOL. WDYT?

**Comment by ericvn** [comment]
LGTM

**Review by dhawton — APPROVED** [review]
works for me, thanks!

**Pull request merged**